### PR TITLE
My pets on profile

### PR DESCRIPTION
**Pull request by randzoh**
- "My profile" brings the user to their profile instead of the edit form
- The user's pets are now shown in the user's profile under their own info.
- They can edit their pets from the list.
- They can delete the pet from the edit form.

### CODE REVIEW

File: app/views/shared/_pet_form.html.erb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+ <%= simple_form_for [ @user, @pet ] do |f| %>
+      <%= f.input :name %>
+      <%= f.input :species %>
+      <%= f.input :breed %>
+      <%= f.input :photo, as: :file %>
```
Comment (wendy-ph) [inline app/views/shared/_pet_form.html.erb:5]: In Pet model, there's no attached association ("has_one_attached :photo" or "has_many_attached :photos")

**Comment by wendy-ph** [comment]
can you add a 'Back' link that will bring us to the my-profile page?

File: app/views/shared/_pet_form.html.erb (hunk 1 above)
Comment (randzoh) [reply app/views/shared/_pet_form.html.erb:5]: I'll add "has_one_attached :photo" for now and we can change it when we implement Cloudinary if we want multiple photos

Comment (randzoh) [reply app/views/shared/_pet_form.html.erb:5]: I'll actually comment out the photo input & take care of all this in the Cloudinary PR. I'll make a trello card for it.

**Pull request merged**